docusaurus.config.js

```diff
@@ -3,42 +3,47 @@ const darkCodeTheme = require('prism-react-renderer/themes/vsDark');
 
 /** @type {import('@docusaurus/types').DocusaurusConfig} */
 module.exports = {
-  title: 'IPC144 - Course Notes',
-  tagline: 'C Programming',
-  url: 'https://cghub.ca',
-  baseUrl: '/',
+  title: "IPC144 - Course Notes",
```

Comment:
Please only change the parts of this file that you need to, don't reformat everything.

Reply:
ok